src/test_driver/linux-cirque/MobileDeviceTest.py

```diff
@@ -97,7 +97,7 @@ def run_controller_test(self):
         self.execute_device_cmd(req_device_id, "pip3 install {}".format(os.path.join(
             CHIP_REPO, "out/debug/linux_x64_gcc/controller/python/chip_repl-0.0-py3-none-any.whl")))
 
-        command = "gdb -return-child-result -q -ex run -ex bt --args python3 {} -t 240 -a {} --paa-trust-store-path {}".format(
```

Comment:
Artificially lowered the timeout to induce a failure in CI to ensure this is all behaving correctly.